manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.installer.yaml

```diff
@@ -0,0 +1,12 @@
+PackageIdentifier: HoYoverse.GenshinImpact
+PackageVersion: 2.25.2.0
+Installers:
+- InstallerUrl: https://sg-public-api.hoyoverse.com/event/download_porter/link/ys_global/genshinimpactpc/default
+  Architecture: x64
+  InstallerType: exe 
+  InstallerSwitches:
+    Silent: n/a
+    SilentWithProgress: n/a
```

Comment:
These switches don't appear to give an unattended install. The install must be able to be run unattended in order for the package to be accepted

![image](https://user-images.githubusercontent.com/12611259/232563218-f25ef68e-72a7-43f5-ac5d-eb8f79c84e49.png)